niceman/distributions/conda.py

```diff
@@ -345,19 +363,18 @@ def identify_distributions(self, paths):
                 packages.append(package)
 
             # Give the distribution a name
-            if (len(conda_paths)) > 1:
-                env_name = 'conda_env-%d' % idx
+            # Determine name from path (Alt approach: use conda-env info)
+            if os.path.normpath(conda_path) == os.path.normpath(root_path):
+                env_name = "root"
             else:
-                env_name = 'conda_env'
+                env_name = os.path.basename(os.path.normpath(conda_path))
```

Comment:
I'm guessing the `normpath` call is to deal with trailing slashes so that "base/" still returns "base"?

Reply:
Yes, I was afraid that the paths given to us could have lots of weirdness (like ../ or ending slashes), so by normalizing I can make it more resilient.

Reply:
Makes sense, and certainly helps for the trailing slash.  Not that it matters, but I wasn't able to think up a way where `basename` would return a different value because of "../" or any other weirdness in the path leading to the base name.